proto/cosmos/auth/query.proto

```diff
@@ -11,10 +12,14 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
 // Query creates service with Account and Parameters as rpc
 service Query{
     // Account returns account details based on address
-    rpc Account (QueryAccountRequest) returns (QueryAccountResponse) {}
+    rpc Account (QueryAccountRequest) returns (QueryAccountResponse) {
+      option (google.api.http).get = "/cosmos/auth/v1beta/accounts/{address}";
+    }
 
     // Params queries all parameters
-    rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {}
+    rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
+      option (google.api.http).get = "/cosmos/auth/v1beta/params";
```

Comment:
Are we adopting `v1beta` or `v1beta1`? @amaurymartiny ?

Reply:
This is waiting on #6905

Reply:
Yeah, I had put a note at https://github.com/cosmos/cosmos-sdk/pull/6918#discussion_r465701144. As per last Friday's discussion, it should be safe to start changing the path to `v1beta1` everywhere here.